src/components/Fields.tsx

```diff
@@ -36,46 +36,49 @@ import {
   TableRow,
   TableBody,
   theme
-} from "@looker/components";
-import styled from "styled-components";
-import {ColumnDescriptor, CommentPermissions} from "./interfaces";
-import {ILookmlModel, ILookmlModelExplore, ILookmlModelExploreField, IUser} from "@looker/sdk";
-import { DetailDrawer } from "./DetailDrawer";
-import { DETAILS_PANE } from "../utils/constants";
+} from "@looker/components"
+import styled from "styled-components"
+import { ColumnDescriptor, CommentPermissions } from "./interfaces"
+import {
+  ILookmlModel,
+  ILookmlModelExplore,
+  ILookmlModelExploreField,
+  IUser
+} from "@looker/sdk"
+import { DetailDrawer } from "./DetailDrawer"
+import { DETAILS_PANE } from "../utils/constants"
 
-// @ts-ignore
-export const TableWrapper = styled(Box)`
-  border-bottom: .5px solid ${theme.colors.ui2};
+export const TableWrapper = styled(Box as any)`
+  border-bottom: 0.5px solid ${theme.colors.ui2};
 
   &:last-child {
     border-bottom: none;
   }
-`;
+`
 
 // Sticky Table Header
-// @ts-ignore
-export const StickyHeader = styled(TableHeaderCell)`
+export const StickyHeader = styled(TableHeaderCell as any)`
```

Comment:
There are a few things Prettier wouldn't do, but recommended I do. Removing ts-ignore and adding "as any" was something I did manually